OSDOCS-17075_c#Updating clusters CQA (#115160)
openshift/openshift-docs: modules/update-best-practices.adoc

[id="update-best-practices_{context}"]
= Best practices for cluster updates

[role="_abstract"]

> Follow best practices to ensure successful cluster updates, including selecting recommended versions, resolving critical alerts, maintaining spare node capacity, and properly configuring pod disruption budgets

Reviewer: ```suggestion
Follow best practices to ensure successful cluster updates. These best practices include selecting recommended versions, resolving critical alerts, maintaining spare node capacity, and properly configuring pod disruption budgets.
```

> {product-title} provides a robust update experience that minimizes workload disruptions during an update. Updates will not begin unless the cluster is in an upgradeable state at the time of the update request.

Reviewer: ```suggestion
{product-title} minimizes workload disruptions during an update. Updates do not begin unless the cluster is in an upgradeable state at the time of the update request.
```
no future tense; apply minimalism

The OpenShift Update Service (OSUS) provides update recommendations based on cluster characteristics such as the cluster's subscribed channel.
The Cluster Version Operator saves these recommendations as either recommended or conditional updates.

While it is possible to attempt an update to a version that is not recommended by OSUS, following a recommended update path protects users from encountering known issues or unintended consequences on the cluster.

Choose only update targets that are recommended by OSUS to ensure a successful update.

[id="critical-alerts_{context}"]
== Address all critical alerts on the cluster

> Critical alerts must always be addressed as soon as possible, but it is especially important to address these alerts and resolve any problems before initiating a cluster update. Failing to address critical alerts before beginning an update can cause problematic conditions for the cluster.

Reviewer: ```suggestion
Critical alerts must always be addressed as soon as possible, but it is especially important to address these alerts and resolve any problems before initiating a cluster update.

Failing to address critical alerts before beginning an update can cause problematic conditions for the cluster.
```
i think the original was better; making these paragraphs more dense just results in a wall of text that isn't scannable
